cluster/manifests/01-coredns-local/daemonset-coredns.yaml

```diff
@@ -80,9 +92,14 @@ spec:
           readOnly: true
           subPath: unbound.conf
       - name: unbound-telemetry
+{{- if eq .Cluster.Provider "zalando-eks" }}
+        image: 926694233939.dkr.ecr.eu-central-1.amazonaws.com/production_namespace/teapot/unbound-telemetry:master-5
+{{- else }}
         image: container-registry.zalando.net/teapot/unbound-telemetry:master-5
+{{- end }}
         args:
         - tcp
+        # TODO: ipv6
         - --bind=0.0.0.0:9054
```

Comment:
Should this bind to all interfaces (for IPv6)?

Reply:
This is unbound which I didn't invest in making ipv6 compatible yet. We use dnsmasq in eks clusters for now so we can defer fixing this to later.